.gitlab-ci.yaml

```diff
@@ -0,0 +1,45 @@
+stages:          # List of stages for jobs, and their order of execution
+  - setup
+  - package
+  - release
+
+variables:
+  GITHUB_BRANCH_NAME: ${CI_COMMIT_BRANCH} # Redefining the variables that are replicated-cli specifics
+  GITHUB_TAG_NAME: ${CI_COMMIT_TAG} # Redefining the variables that are replicated-cli specifics
+
+'Verify Variables': # Added this stage just to verify that all variables are set as expected
+    stage: setup
+    script:
+      - "echo Release branch name: ${GITHUB_BRANCH_NAME}"
+      - "echo Release tag name: ${GITHUB_TAG_NAME}"
+      - "echo Replicated Application: ${REPLICATED_APP}" # This variable is set as a Project CI/CID Variable to not expose the value in the source code
+      - "echo Replicated API Token ${REPLICATED_API_TOKEN}" # This variable is set as a Project CI/CID Variable to not expose the value in the source code
```

Comment:
will this print the token in build logs?

Reply:
Yes, it may print it if they're not masked. This is meant for debugging purposes what's explicitly mentioned in the comments.

Want me to remove this?